src/shutdown/umount.c

```diff
@@ -155,6 +160,22 @@ int mount_points_list_get(const char *mountinfo, MountPoint **head) {
                 if (!m)
                         return log_oom();
 
+                iter_children = mnt_new_iter(MNT_ITER_FORWARD);
+                if (!iter_children)
+                        return log_oom();
+
+                struct libmnt_fs *child;
+                r = mnt_table_next_child_fs(table, iter_children, fs, &child);
+                if (r < 0) {
+                        return log_error_errno(r, "Failed to get children mounts for %s from %s: %m", path, mountinfo ?: "/proc/self/mountinfo");
+                } else if (r == 1) {
+                        leaf = true;
+                } else {
+                        leaf = false;
+                }
+                /* We care only whether it exists, it is unused */
+                (void)child;
```

Comment:
Hmm, what is this code doing? If this is about silencing of a warning about an unused variable, put `_unused_` on the declaration instead.

Reply:
Correct. Done.